packages/@okta/vuepress-site/docs/concepts/policies/index.md

```diff
@@ -4,116 +4,155 @@ meta:
 
-* [Authentication policy](/docs/reference/api/policy/#authentication-policy): Determines the extra levels of authentication that you want performed before a user can access an application, such as enforcing factor requirements. Every app in your org has one authentication policy, and multiple apps can share a policy. Okta provides some preset policies with standard sign-on requirements, including a default policy automatically assigned to new apps. The default policy allows access with any two factors.
+When you want to restrict access to an API based on the calling application, you can create an [access policy](/docs/reference/api/authorization-servers/#policy-object).
 
-  You can create an authentication policy specifically for the app or create a few policies and [share them](https://help.okta.com/okta_help.htm?type=oie&id=ext-share-auth-policy) across multiple apps. If you decide later to change an app's sign-on requirements, you can modify its policy or switch to a different policy using the [Authentication Policies page](https://help.okta.com/okta_help.htm?type=oie&id=ext-create-auth-policy). See [Configure a global session policy and an authentication policy](/docs/guides/configure-signon-policy/main/). Additionally, you can [merge duplicate authentication policies with identical rules](https://help.okta.com/okta_help.htm?type=oie&id=ext-merge-auth-policies) to simplify policy management.
+## How policies work
 
-### Password policy
+Policy settings for a particular [policy type](#policy-types) consist of one or more Policy objects with one or more policy rules:
 
-* [Password policy](/docs/reference/api/policy/#password-policy): Determines the requirements for a user's password length and complexity, as well as how often a user must change their password. Okta provides a default policy to enforce the use of strong passwords, but you can also create additional policies that are less or more restrictive and apply them to users based on group membership. This policy also governs the recovery operations that the user may perform, including change password, reset (forgot) password, and self-service password unlock.
+* **Policies** contain groups of resources requiring similar treatment, like apps with the same security characteristics or user groups with the same account setup requirements
+* **Rules** describe the conditions of policy behavior, such as requests from a geographical location or if the user is on a trusted network.
 
-### Enrollment policies
+The conditions of a policy and a rule combined determine whether to apply a policy. As a best practice, restrictive rules should be placed at the top of the priority list. Beyond that, you can create combinations of conditions for multiple scenarios. There’s no limit to the number of rules your policies can have.
```

Comment:
```suggestion
The policy's conditions and rules determine when the policy is applied. As a best practice, restrictive rules should be placed at the top of the priority list. Otherwise, you can create combinations of conditions for different scenarios. There’s no limit to the number of rules your policies can have.
```